src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+    private String color;
+
+    public Circle(int radius, String color) {
+        this.radius = radius;
+        this.color = color;
+    }
+
+    public Circle(int radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public int getArea() {
+        double a = Math.PI * radius * radius;
+        return (int)a;
```

Comment:
```suggestion
        return Math.PI * radius * radius;
```
the same for other figures